src/jsoncharutils.h

```diff
@@ -11,7 +11,7 @@ namespace simdjson {
 
 // these are the chars that can follow a true/false/null or number atom
 // and nothing else
-const uint32_t structural_or_whitespace_or_null_negated[256] = {
+static const uint32_t structural_or_whitespace_or_null_negated[256] = {
```

Comment:
constexpr question here too. Same comments as numberparsing.h.

Reply:
Yes, constexpr works, I tried it. I think that's what it should be.